src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -3146,11 +3044,11 @@ namespace HVACMultiSpeedHeatPump {
         // if the DX heating coil cannot meet the load, trim with supplemental heater
         // occurs with constant fan mode when compressor is on or off
         // occurs with cycling fan mode when compressor PLR is equal to 1
-        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump;
+        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump(MSHeatPumpNum);
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:3149]:(style),[constVariable],Variable 'MSHeatPump' can be declared as reference to const